resources/lib/service.py

```diff
@@ -6,24 +6,43 @@
 
 from __future__ import absolute_import, division, unicode_literals
 from xbmc import Monitor
+from apihelper import ApiHelper
+from favorites import Favorites
 from kodiwrapper import KodiWrapper
+from playerinfo import PlayerInfo
+from resumepoints import ResumePoints
+from statichelper import play_url_to_id, to_unicode, url_to_episode
 from tokenresolver import TokenResolver
-from statichelper import to_unicode
 
 
 class VrtMonitor(Monitor):
     ''' This is the class that monitors Kodi for the VRT NU video plugin '''
 
     def __init__(self):
         ''' VRT Monitor initialisiation '''
+        self._kodi = KodiWrapper(None)
+        self._resumepoints = ResumePoints(self._kodi)
+        self._container = None
+        if self._resumepoints.is_activated():
+            self._playerinfo = PlayerInfo(info=self.handle_info)
+            self._favorites = Favorites(self._kodi)
+            self._apihelper = ApiHelper(self._kodi, self._favorites, self._resumepoints)
         Monitor.__init__(self)
+
+    def run(self):
+        ''' Main loop '''
         while not self.abortRequested():
             if self.waitForAbort(10):
                 break
 
-    @staticmethod
-    def onNotification(sender, method, data):  # pylint: disable=invalid-name
+    def onNotification(self, sender, method, data):  # pylint: disable=invalid-name
         ''' Handler for notifications '''
+        self._kodi.log(2, '[Notification] sender={sender}, method={method}, data={data}', sender=sender, method=method, data=to_unicode(data))
```

Comment:
I expect this one to log lots of unrelated stuff as well ;-)

Reply:
I still have to remove and edit some of the log calls in this branch. I think it can be useful while testing.

Reply:
Sure.